regtests/run.sh

```diff
@@ -20,7 +20,7 @@
 # Run without args to run all tests, or single arg for single test.
 
 export SPARK_VERSION=spark-3.5.2
-export SPARK_DISTRIBUTION=${SPARK_VERSION}-bin-hadoop3-scala2.13
+export SPARK_DISTRIBUTION=${SPARK_VERSION}-bin-hadoop3
```

Comment:
If there is no particular reason to use scala 2.13, I would stick to the default one, which is 2.12.